Dockerfile

```diff
@@ -2,17 +2,18 @@ FROM node
 
 MAINTAINER Michael R. Bernstein
 
-RUN useradd -u 9000 -r -s /bin/false app
+WORKDIR /usr/src/app/
+
+COPY engine.json /usr/src/app/
```

Comment:
ah cool, so `RUN useradd -u 900 -r -s /bin/false app/` ?

Why is that a little different from https://github.com/codeclimate/codeclimate-rubocop/blob/master/Dockerfile#L11 ?